bio_curve_fit/logistic.py

```diff
@@ -321,13 +321,32 @@ def predict_inverse(self, y):
         But for samples of unknown concentration, we want to get the concentration as given
         response, which is what this function does.
 
+        args:
+        y: float
+            The response value for which the corresponding x-value will be calculated.
+        enforce_detection_limits: bool
+            If True, return np.nan for y-values above the maximum asymptote (D) of the curve, and 0 for y-values below the minimum asymptote (A) of the curve.
+
         """
         self._check_fit()
+        if isinstance(y, list):
```

Comment:
can force y to array of floats by something like (requires testing):

y = np.array([float(y)]) if isinstance(int, float) else np.array(y)